python/src/KarhunenLoeveAlgorithmImplementation_doc.i.in

```diff
@@ -11,7 +11,9 @@ s : float, :math:`\geq0`
 
 Notes
 -----
-The Karhunen-Loeve decomposition enables to build some finite approximations of stochastic processes which are optimal with respect to the norm :math:`L^2`.
+The Karhunen-Loeve decomposition enables to build some finite approximations
+of stochastic processes which are optimal with respect to the norm
+:math:`L^2` (see [sullivan2015]_ page 223).
 
 We suppose that :math:`C:\cD \times \cD \rightarrow  \cS^+_d(\Rset)` is a covariance function defined on :math:`\cD \in \Rset^n`, continuous at :math:`\vect{0}`. 
 
```

Comment:
Suggested change below:

"They form a Hilbertian basis of..." (instead of "an hilbertian")

Reply:
"the covariance function C writes" --> "the covariance Function C can be written"

Reply:
"Thus if threshold is set to 0" --> "Thus if the threshold s is set to 0"

Reply:
"The approximated Fredholm problem writes" --> "The approximated Fredholm problem is"

Reply:
"The Fredhold problem writes:" --> "The Fredhold problem can be expressed as:"

Reply:
Ok. Fixed at 5372f34cf91a1357b4e1e45f686865ca6c9a818e